languages/header-es.html

```diff
@@ -1,819 +1,1622 @@
-                              </li>
-                            </ul>
-                          </div>
+            <nav id="navmenu-desktop" class="m-menu--desktop" role="tablist">
+                <a class="m-menu--desktop__toggle collapsed" href="/menu#Transit-section" role="tab" data-toggle="collapse" data-parent="#navmenu-desktop" data-target="#transit" aria-expanded="false" aria-controls="transit">
+                    <font style="vertical-align: inherit;">
+                        <font style="vertical-align: inherit;" class="">
+                            Tránsito </font>
+                    </font>
+                </a>
+                <div class="m-menu--desktop__menu" style="top: 72px;">
+                    <div class="container">
+                        <div class="panel">
+                            <div class="collapse" id="transit" role="tabpanel" aria-expanded="false" style="height: 0px;">
+                                <div class="m-menu-desktop__section transit">
+                                    <div class="m-menu__section-heading">
+                                        <font style="vertical-align: inherit;">
+                                            <font style="vertical-align: inherit;">
+                                                Modos de Tránsito </font>
+                                        </font>
+                                    </div>
+                                    <nav class="m-menu-desktop__submenu-section" aria-label="Modos de Tránsito">
+                                        <a class="m-menu__link" href="/schedules/subway">
+                                            <div>
+                                                <font style="vertical-align: inherit;">
+                                                    <font style="vertical-align: inherit;" class="">Subterraneo</font>
+                                                </font>
+                                            </div>
+                                        </a><a class="m-menu__link" href="/schedules/bus">
+                                            <div>
+                                                <font style="vertical-align: inherit;">
+                                                    <font style="vertical-align: inherit;">Autobús</font>
+                                                </font>
+                                            </div>
+                                        </a><a class="m-menu__link" href="/schedules/commuter-rail">
+                                            <div>
+                                                <font style="vertical-align: inherit;">
+                                                    <font style="vertical-align: inherit;">Tren de cercanías</font>
+                                                </font>
+                                            </div>
+                                        </a><a class="m-menu__link" href="/schedules/ferry">
```

Comment:
**issue:** This removes some of the modifications we made to the links -- these links are missing the `rel` attribute and should be updated to be absolute rather than relative links.